single-pool/program/src/error.rs

```diff
@@ -127,7 +127,7 @@ impl PrintProgramError for SinglePoolError {
                 msg!("Error: Not enough stake to cover the provided quantity of pool tokens. \
                      (Generally this should not happen absent user error, but may if the minimum delegation increases.)"),
             SinglePoolError::SignatureMissing => msg!("Error: Required signature is missing."),
-            SinglePoolError::WrongStakeState => msg!("Error: Stake account is not in the state expected by the program."),
+            SinglePoolError::WrongStakeStateV2 => msg!("Error: Stake account is not in the state expected by the program."),
```

Comment:
also here (github suggestions doesnt let me fix this...)

Reply:
actually ill just push my own commit, need to handle merge conflicts anyway

Reply:
Thanks for fixing this!